### PR TITLE
docs(mixed): add gender other to gender selectors

### PR DESCRIPTION
**Pull request by tanohzana**
I added a gender neutral option in the selectors in examples as I felt some communities might be impacted by the binary choice here.

### CODE REVIEW

File: src/lib/SUI.js [hunk 1]
```diff
@@ -609,6 +609,7 @@ export const GENDERS = [
   'mars stroke vertical',
   'mercury',
   'neuter',
+  'other',
```
Comment (layershifter) [inline src/lib/SUI.js]: This array contains definitions for existing icons, please revert this change.

Comment (tanohzana) [reply src/lib/SUI.js]: ff96fdc should fix it
